src/errors.ts

```diff
@@ -1,72 +0,0 @@
-import Ajv, { AdditionalPropertiesParams, EnumParams, RequiredParams } from "ajv";
```

Comment:
why remove this ?

Reply:
no longer used as we are not validating the passed in commands now. It's not used in any of the other implementations as well

Reply:
moved it within wrap